platform/admin/core/admin-core-logviewer/src/main/webapp/js/components/log-entry/log-entry.js

```diff
@@ -46,20 +49,37 @@ export default ({ entry, marks }) => {
     }
   }
 
+  const expandEntry = () => {
+    dispatch(actions.expandEntry(entry.hash))
+  }
+
+  const getMessageClasses = () => {
+    return (entry.hash === expandedHash) ? 'rowData messageExpanded' : 'rowData message'
+  }
+
+  const getRowClasses = () => {
+    if (entry.hash === expandedHash) {
+      return 'rowAnimation selectedRow'
+    } else {
+      return 'rowAnimation'
+    }
+  }
+
   return (
-    <tr className='rowAnimation'>
-      <td className='row' style={{ background: levels(entry.level) }} width={175}>
+    <tr onClick={expandEntry} className={getRowClasses()} style={{ background: levels(entry.level) }}>
```

Comment:
Instead of using a `style` attribute here, could we pass in the level to `getRowClasses()` and have it return a class based on the level? Not sure that someone looking to change the colors associated with log levels would think of looking inside a js file to do that. More logical to change a less/css file.